ethcore/engines/clique/src/lib.rs

```diff
@@ -571,7 +570,7 @@ impl Engine for Clique {
 
 		// Don't waste time checking blocks from the future
 		{
-			let limit = CheckedSystemTime::checked_add(SystemTime::now(), Duration::from_secs(self.period))
+			let limit = SystemTime::now().checked_add(Duration::from_secs(self.period))
```

Comment:
here it doesn't matter if we're not taking 2038 problem into account as `self.period` is a config param and if it's misconfigured, it's not going to work anyway